doc/sources/array_api.rst

```diff
@@ -0,0 +1,120 @@
+.. ******************************************************************************
+.. * Copyright 2024 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _array_api:
+
+=================
+Array API support
+=================
+The `Array API <https://data-apis.org/array-api/latest/>`_ specification defines
+a standard API for all array manipulation libraries with a NumPy-like API.
+Intel(R) Extension for Scikit-Learn doesn't require
+`array-api-compat <https://github.com/data-apis/array-api-compat>`__ to be installed for
+functional support of the array API standard.
+In the current implementation, the functional support of array api follows the functional
+support of different array or DataFrame inputs and does not modify the precision of the input and
+output data formats unless necessary. Any array API input data provided will be converted to host
+numpy.ndarrays and all internal manipulations with data will be done with these representations of
+the input data. DPNP's 'ndarray' and Data Parallel Control's 'usm_ndarray' have special handling
+requirements that are described in the relevant section of this document. Output values will in
+all relevant cases match the input data format.
+
+.. note::
+    Currently, only `array-api-strict <https://github.com/data-apis/array-api-strict>`__,
+    `dpctl <https://intelpython.github.io/dpctl/latest/index.html>`__, `dpnp <https://github.com/IntelPython/dpnp>`__
+    and `numpy <https://numpy.org/>`__ are known to work with sklearnex estimators.
+.. note::
+    Stock Scikit-learn’s array API support requires `array-api-compat <https://github.com/data-apis/array-api-compat>`__ to be installed.
+
+
+Support for DPNP and DPCTL
+==========================
+The functional support of input data for sklearnex estimators also extended for non-array API
+compliant array formats not covered by array-api-compat. These include SYCL USM arrays
```

Comment:
```suggestion
The functional support of input data for sklearnex estimators also extended for SYCL USM array types. These include
```

Reply:
It contradicts with the previous suggestion, but ok will apply :)